src/generateProject/generationWizard.ts

```diff
@@ -26,11 +26,31 @@ import { validateArtifactId, validateGroupId, validatePackageName, validateResou
 export async function generateProjectWizard() {
 
   const state: Partial<ProjectGenState> = {
-    totalSteps: 6
+    totalSteps: 7
   };
 
   async function collectInputs(state: Partial<ProjectGenState>) {
-    await MultiStepInput.run(input => inputGroupId(input, state));
+    await MultiStepInput.run(input => inputBuildTool(input, state));
+  }
+
+  async function inputBuildTool(input: MultiStepInput, state: Partial<ProjectGenState>) {
+
+    interface BuildToolPickItem extends QuickPickItem {
+      value: string;
+    }
+
+    const quickPickItems = [{label: 'Maven', value: 'MAVEN'}, {label: 'Gradle', value: 'GRADLE'}];
+
```

Comment:
build tool choice should be remembered

Reply:
After the first time the user selects a build tool, it saves the choice so next time, the build tool selection step is skipped? And then to change the build tool, maybe add a vscode setting?

Reply:
no, just preselect it, keep it displayed